packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -153,7 +152,7 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         address _guardian,
         bool _paused,
         SystemConfig _config
-    ) Semver(1, 3, 0) {
+    ) Semver(1, 3, 1) {
```

Comment:
Do we update semvers on comments too?

Reply:
Yup
https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts-bedrock#versioning